utils/data_augumentation.py

```diff
@@ -85,7 +85,7 @@ def __call__(self, image, boxes=None, labels=None):
 
 
 class ToAbsoluteCoords(object):
-    def __call__(self, image, boxes=None, labels=None):
+    def __call__(self, image, boxes, labels):
```

Comment:
remove default `None` value because boxes and labels with `None` cannot be handled properly in this method